rllib/agents/pg/torch_pg_policy.py

```diff
@@ -3,38 +3,37 @@
 from __future__ import print_function
 
 import ray
-from ray.rllib.evaluation.postprocessing import compute_advantages, \
-    Postprocessing
+from ray.rllib.agents.pg.tf_pg_policy import post_process_advantages
+from ray.rllib.evaluation.postprocessing import Postprocessing
 from ray.rllib.policy.sample_batch import SampleBatch
 from ray.rllib.policy.torch_policy_template import build_torch_policy
+from ray.rllib.utils.framework import try_import_torch
 
+torch, _ = try_import_torch()
 
-def pg_torch_loss(policy, model, dist_class, train_batch):
+
+def torch_pg_loss(policy, model, dist_class, train_batch):
+    """The basic policy gradients loss."""
     logits, _ = model.from_batch(train_batch)
     action_dist = dist_class(logits, model)
     log_probs = action_dist.logp(train_batch[SampleBatch.ACTIONS])
-    # save the error in the policy object
-    policy.pi_err = -train_batch[Postprocessing.ADVANTAGES].dot(
-        log_probs.reshape(-1))
+    # Save the error in the policy object.
+    # policy.pi_err = -train_batch[Postprocessing.ADVANTAGES].dot(
+    # log_probs.reshape(-1)) / len(log_probs)
```

Comment:
Remove this commented out code?

Reply:
Was thinking about making it optional to use either the sum or the mean.